beacon-chain/slasher/service.go

```diff
@@ -39,6 +39,8 @@ type ServiceConfig struct {
 	SlashingPoolInserter    slashings.PoolInserter
 	HeadStateFetcher        blockchain.HeadFetcher
 	SyncChecker             sync.Checker
+	ClockWaiter             startup.ClockWaiter
+	ClockSetter             startup.ClockSetter
```

Comment:
why both ? slasher should only need a waiter

Reply:
Thanks for the catch, I *think* it was previously needed for a test but after refreshing from develop I can't find that code, removing.